### ISSUE TITLE
Update Navigator map polygons to use latest EarthByte model

### ISSUE THREAD

**Issue by aazaff** [paleobiodb/navigator#27]
Update the navigator map polygons to use the same rotation model as occurrences and collections.

**Comment by aazaff** [comment]
Does anyone know who is technically responsible for this - i.e., who should be assigned to this issue?

**Comment by jczaplew** [comment]
Instead of caching and loading static files for the plates, we should hit the Macrostrat API `https://macrostrat.org/api/v2/paleogeography?age=<midpoint of target interval>&format=topojson_bare`

**Comment by aazaff** [comment]
Agreed, suggest we fix this as part of the proposed major paleogeography refactor #72
